pkg/lvm/lvm.go

```diff
@@ -253,6 +256,82 @@ func umountLV(targetPath string) {
 	}
 }
 
+func createGetCapacityPod(ctx context.Context, va volumeAction) (int64, error) {
+	pvCapacityRegexp, err := regexp.Compile(pvCapacityRegexpString)
+	if err != nil {
+		klog.Errorf("unable to compile regexp for querying physical volume capacity regexp:%s err:%v", pvCapacityRegexpString, err)
+		return 0, err
+	}
+
+	// Wrap command in a shell or the device pattern is wrapped in quotes causing pvdisplay to error
+	provisionerPod, deleteFunc, err := createPod(
+		ctx,
+		"sh",
+		[]string{"-c", fmt.Sprintf("pvdisplay %s %s %s", va.devicesPattern, "--units=B", "-C")},
```

Comment:
IMHO it would be much better to execute

```
 pvs --reportformat json
```

and parse the output into a go struct instead of using regex